packages/nodes-base/nodes/Xero/ContactDescription.ts

```diff
@@ -250,66 +250,66 @@ export const contactFields = [
 				default: '',
 				description: 'Last name of contact person (max length = 255)',
 			},
-			// {
-			// 	displayName: 'Phones',
-			// 	name: 'phonesUi',
-			// 	type: 'fixedCollection',
-			// 	typeOptions: {
-			// 		multipleValues: true,
-			// 	},
-			// 	default: '',
-			// 	placeholder: 'Add Phone',
-			// 	options: [
-			// 		{
-			// 			name: 'phonesValues',
-			// 			displayName: 'Phones',
-			// 			values: [
-			// 				{
-			// 					displayName: 'Type',
-			// 					name: 'type',
-			// 					type: 'options',
-			// 					options: [
-			// 						{
-			// 							name: 'Default',
-			// 							value: 'DEFAULT',
-			// 						},
-			// 						{
-			// 							name: 'DDI',
-			// 							value: 'DDI',
-			// 						},
-			// 						{
-			// 							name: 'Mobile',
-			// 							value: 'MOBILE',
-			// 						},
-			// 						{
-			// 							name: 'Fax',
-			// 							value: 'FAX',
-			// 						},
-			// 					],
-			// 					default: '',
-			// 				},
-			// 				{
-			// 					displayName: 'Number',
-			// 					name: 'phoneNumber',
-			// 					type: 'string',
-			// 					default: '',
-			// 				},
-			// 				{
-			// 					displayName: 'Area Code',
-			// 					name: 'phoneAreaCode',
-			// 					type: 'string',
-			// 					default: '',
-			// 				},
-			// 				{
-			// 					displayName: 'Country Code',
-			// 					name: 'phoneCountryCode',
-			// 					type: 'string',
-			// 					default: '',
-			// 				},
-			// 			],
-			// 		},
-			// 	],
-			// },
+			{
+				displayName: 'Phones',
```

Comment:
Same thing here. You don't need these objects 2 times, you can just define it once and set the operation as create and update.

Reply:
@Jhalter5Stones same as my comment above